README.md

```diff
@@ -52,6 +52,14 @@ chmod +x minio
 ./minio server /data
 ```
 
+### iptables in RHEL/CentOS
```

Comment:
Could we also add `ufw` i.e Ubuntu/Debian as well? - this section can be a firewall not specific to a platform.

Reply:
good idea. will do

Reply:
good idea. I concur with harsha. I love digital oceans guides on these topics.